neural_compressor/algorithm/smooth_quant.py

```diff
@@ -46,6 +46,7 @@ def __init__(self, alpha=0.5):
         # scales_per_op: True, each op will have an individual scale, mainly for accuracy
         #                False, ops with the same input will share a scale, mainly for performance
         self.alpha = alpha
+        self.folding = False
```

Comment:
Can we add a descriptor for this argument?

Reply:
I mean we should add a descriptor in docs.

Reply:
I will add document in next PR.